docs/building-deployer.md

```diff
@@ -37,10 +37,10 @@ and test your application. You can use it to inspect your development
 environment, test your application's installation, and run smoke tests
 on your application.
 
-[Review the prerequisites, and install the development tool](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/tool-prerequisites.md).
+[Review the prerequisites, and install the development tool](/tool-prerequisites.md).
```

Comment:
The leading slashes have to be removed from these links (you should be able to preview this file; there, these links result in 404 as they escape the `docs` folder.

Here and other places.